perl/CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+Please read [CONTRIBUTING](https://github.com/cucumber/gherkin/blob/master/CONTRIBUTING.md) first.
+You should clone the [cucumber/gherkin](https://github.com/cucumber/gherkin) repo if you want
+to contribute.
+
+You will need `cpanm` installed on your system
```

Comment:
I had some questions about this...

The tests rely on being able to find the parent test-data stuff, which - I think - makes a perl/.travis file unfeasible. Should I simply add to the main project travis file? The cpanm dep is easy to add to travis